formidable/__init__.py

```diff
@@ -1,3 +1,20 @@
-default_app_config = 'formidable.app.FormidableConfig'
+import warnings
+
 
+default_app_config = 'formidable.app.FormidableConfig'
 version = '0.15.0.dev0'
+
+# TO BE REMOVED ON 1.0.0
+warnings.simplefilter('always', DeprecationWarning)
+warnings.warn(
+    'Form Presets are about to be deprecated. '
+    'Please refer to the Deprecation Timeline document for more details '
+    'https://django-formidable.readthedocs.io/en/latest/deprecations.html.',
+    DeprecationWarning)
+warnings.warn(
+    'Support for Django Rest Framework v3.3 is about to be deprecated. '
+    'Please refer to the Deprecation Timeline document for more details '
+    'https://django-formidable.readthedocs.io/en/latest/deprecations.html.',
+    DeprecationWarning
+)
+warnings.simplefilter('ignore', DeprecationWarning)
```

Comment:
Not sure about this. I'd use `'default'` or maybe [warnings.catch_warnings](https://docs.python.org/2/library/warnings.html#warnings.catch_warnings) contextmanager.

Reply:
ah, yes, the context manager, good idea!